examples/machine-controller.yaml

```diff
@@ -24,6 +24,38 @@ subjects:
   kind: Group
   name: system:bootstrappers:machine-controller:default-node-token
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: machine-controller:kubelet-config-1.11
+  namespace: kube-system
+rules:
+- apiGroups:
+  - ""
+  resourceNames:
+  - kubelet-config-1.11
```

Comment:
if you don't mind, could you explain why we need to define this role ?

Reply:
Becuase kubeadm 1.11 uses dynamic kubelet config which means the kubelet gets its config from a configmap. This behaviour is not configurable: https://github.com/kubernetes/kubernetes/blob/3efd75d061f250108ca5cb5a9cc088518fb03676/cmd/kubeadm/app/cmd/join.go#L286-L289